tests/conftest.py

```diff
@@ -28,6 +28,7 @@
 
 import pytest
 import requests
+from milvus import default_server
```

Comment:
Lets move this import inside of `milvus_server_uri` so its only imported if needed.